apps/src/templates/sectionsRefresh/coteacherSettings/AddCoteacher.jsx

```diff
@@ -61,6 +110,10 @@ export default function AddCoteacher({
     [handleAddEmail]
   );
 
+  const isMaxCoteachers = useCallback(() => {
+    return numCoteachers >= 5;
+  }, [numCoteachers]);
```

Comment:
Is a callback actually useful for such a simple computation?

```suggestion
  const isMaxCoteachers = numCoteachers >= 5;
```